morpheus/stages/boundary/linear_boundary_stage.py

```diff
@@ -68,11 +80,23 @@ def _build_single(self, builder: srf.Builder, input_stream: StreamPair) -> Strea
 
 class LinearBoundaryIngressStage(SingleOutputSource):
     """
+    TheLinearBoundaryIngressStage acts as source ingress point from a corresponding egress in another linear segment.
+    Given an existing linear pipeline that we want to connect to another segment, a linear boundary egress stage would
+    be added to it and a matching LinearBoundaryIngressStage would be created to receive the e.
+
     Parameters
     ----------
     c : `morpheus.config.Config`
         Pipeline configuration instance.
-
+    boundary_port_id : `str`
+        String corresponding to the name of the LinearBoundaryEgressStage's egress port, use to identify and connect
+        with the appropriate egress.
+    data_type : `object`
+        Data type that this Stage will accept, which will correspond to some existing egress output.
+
+    Example
+    -------
+        boundary_ingress = LinearBoundaryIngressStage(config, "my_boundary_port", int)
```

Comment:
Same here. Remove this example in favor of one on `LinearPipeline.add_segment_boundary()`